Common/src/gRPC/Services/Agent.cs

```diff
@@ -428,8 +453,9 @@ await foreach (var request in requestStream.ReadAllAsync(cancellationToken)
                                           ("taskId", taskData_.TaskId),
                                           ("sessionId", sessionData_.SessionId));
 
-    Task? completionTask = null;
-    var   fsmResult      = new ProcessReplyResultStateMachine(logger_);
+    Task? completionTask       = null;
+    Task? findDependenciesTask = null;
```

Comment:
You can initialize those with `Task.CompletedTask` or `Task.FromCanceled` to avoid the nullable.

Reply:
We will need to properly activate nullables in Core. There are issues with it. Sometimes there are hint saying that nullable are not activated.